lib/logstash/inputs/elasticsearch.rb

```diff
@@ -439,4 +436,42 @@ def parse_user_password_from_cloud_auth(cloud_auth)
     [ cloud_auth.username, cloud_auth.password ]
   end
 
+  module URIOrEmptyValidator
```

Comment:
just to verify: this is an unrelated cleanup right? not problem including it but just wanted to make sure.

Reply:
Yes, that is why it is in a separate commit with a message to that effect.

Reply:
ah, missed that, thanks.